rust-app/src/settings.rs

```diff
@@ -17,13 +17,15 @@ impl Default for Settings {
 impl Settings {
     #[inline(never)]
     pub fn get(&self) -> u8 {
+        #[allow(static_mut_refs)]
```

Comment:
Is this a new lint?

Reply:
"New" since half a year https://github.com/rust-lang/rust/issues/128794